UX-Guide-Metadata/principles/index.html

```diff
@@ -95,7 +95,7 @@ <h2>Discovering Accessible Content</h2>
 				retrieves all books with metadata of <code>&lt;link rel="dcterms:conformsTo"
 					href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a" /&gt;</code> or the
 				equivalent for wcag-aa and ultimately for wcag-aaa would fill this need.</p>
-			<p>Audiobooks created for mainstream use provide important access for many users with disabilities even
+			<p>Full audio books created for mainstream use provide important access for many users with disabilities even
```

Comment:
Just a phrasing nitpick, but even with the extra space it still sort of sounds like a full "audiobook".

Could this maybe be something like: "Books with full audio" ?

It would need changing in a number of other spots if acceptable.

Reply:
Changes made and pushed.